python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -215,6 +215,27 @@ def predict(self, X, output_class=True, algo='auto', threshold=0.5,
         """
         Predicts the labels for X.
 
+        GPU-based prediction in a multi-node, multi-GPU context works
+        by sending the sub-forest from each worker to the client,
+        concatenating these into one forest with the full
+        `n_estimators` set of trees, and sending this combined forest to
+        the workers, which will each infer on their local set of data.
+        Within the worker, this uses the cuML Forest Inference Library
+        (cuml.fil) for high-throughput prediction.
+
+        This allows inference to scale to large datasets, but the forest
+        transmission incurs overheads for very large trees. For inference
+        on small datasets, this overhead may dominate prediction time.
+
+        The 'CPU' fallback method works with sub-forests in-place,
+        broadcasting the datasets to all workers and combining predictions
+        via a voting or averaging method at the end. This method is slower
```

Comment:
wondering if we should just mention `via a voting method at the end` here and just mention `averaging method` for regression.